src/algebra/algebra/tower.lean

```diff
@@ -0,0 +1,292 @@
+/-
+Copyright (c) 2020 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau
+-/
+
+import algebra.algebra.subalgebra
+
+/-!
+# Towers of algebras
+
+In this file we prove basic facts about towers of algebra.
```

Comment:
Could you expand this doc string with more information? Both mathematical information (what is a "tower of algebra," what is this file useful for) and API information (main declarations, implementation details) would be helpful!

Reply:
Done.